### PR TITLE
Rebrand refinement

### PR DESCRIPTION
**Pull request by davidrapson**
Doing one more pass through our components list to bring them in line with our proposed designs.

I'm keen to avoid too many conditionals in our templates to support the rebrand so these changes may result in some minor cosmetic differences to the current live designs. I'll call out where those are so we can make a call on if the changes are acceptable.

### CODE REVIEW

**Comment by davidrapson** [comment]
Removing border from basic card macro

image

**Comment by davidrapson** [comment]
Removing border from media figures

image

**Comment by davidrapson** [comment]
Made segment images flush

image

**Comment by davidrapson** [comment]
Removed border from miniature heroes

image

**Comment by davidrapson** [comment]
Removed colour themes from CMS powered pages (after conversation with @LynseyJReynolds )

image

File: views/includes/metadata.njk [hunk 1]
```diff
@@ -37,7 +37,7 @@
 <meta name="apple-mobile-web-app-title" content="{{ globalCopy.brand.title }}">
 <meta name="application-name" content="{{ globalCopy.brand.title }}">
 
-{% set metaThemeColour = "#ec008c"  %}
+{% set metaThemeColour = "#e5007d" if useNewBrand else "#e20087"  %}
```
Comment (mattandrews) [inline views/includes/metadata.njk]: Good spot 👍

**Pull request merged**